### DISCUSSION TITLE
Termux audio recording eroor

### DISCUSSION THREAD

**Discussion by aniket982321** [termux-play-store/termux-issues#8] [Q&A]
Report issues at https://bugs.termux.net
~ $ termux-microphone-record -l 5 -f test_audio.wav
^C
~ $ ^C
~ $ ls -l -f test_audio.wav
ls: cannot access 'test_audio.wav': No such file or directory
I am getting this type of error please give me solution

**Comment by Djspac3** [comment]
**_I think_** this might be just how it's meant to do. 
**Don't trust me, I'm an idiot**, but seriously, I think that's actually what's meant to do because, why would it save during the recording? And why wouldn't it save when you cancel The action?
